### PR TITLE
update a comment for snapshot example

### PR DESCRIPTION
**Pull request by TangoEnSkai**
I know this is very small thing, but I just wondering this comment is clearer and easier to understand.

### CODE REVIEW

**Comment by abhinav** [comment]
Hey, it looks like there are a bunch of unrelated changes in this commit. You're only trying to change the comment, right? The diff shows a bunch of other things including the TOC. Can you remove those?

**Comment by TangoEnSkai** [comment]
thanks. oh yes, I realise my linter make the links in that tree-structure. let me revise it. sorry.

File: style.md [hunk 1]
```diff
@@ -375,8 +375,8 @@ func (s *Stats) Snapshot() map[string]int {
   return s.counters
 }
 
-// snapshot is no longer protected by the mutex, so any
-// access to the snapshot is racy.
+// snapshot is no longer protected by the mutex,
+// so any access to the snapshot is subject to race condition.
```
Comment (mway) [inline style.md]: spelling: "condition" should be "conditions"

Comment (abhinav) [inline style.md]:
```suggestion
// so any access to the snapshot is subject to race conditions.
```

**Comment by TangoEnSkai** [comment]
thanks for your time!

File: style.md (hunk 1 above)
Comment (prashantv) [reply style.md]: - I think there's no need to move `so any`
 - optionally can call out data races specifically

```
// snapshot is no longer protected by the mutex, so any
// access to the snapshot is subject to data races.
```

Comment (TangoEnSkai) [reply style.md]: I like it. I will update it very soon. thanks

**Comment by TangoEnSkai** [comment]
@prashantv thanks for your comment. just updated as your recommendation.

**Pull request merged**